cloudformation/auxiliary_templates/custom_balancer/README.md

```diff
@@ -0,0 +1,59 @@
+# Deadline Custom Job Balancer
+
+This CloudFormation template deploys a sidecar Lambda function that controls the maxWorkerCount for each job within a Queue to implement functionality similar to the Deadline10 Balancer options.
```

Comment:
> a sidecar Lambda function

nit: Just "a Lambda function" fits well. What does it mean for it to be sidecar vs not?